testsets/smartphone_android.yaml

```diff
@@ -63,3 +63,11 @@
   name: UNKNOWN
   os: Android
   category: smartphone
+
+# Webview + Android(Lollipop and Above)
+- target: 'Mozilla/5.0 (Linux; Android 5.1.1; Nexus 5 Build/LMY48B; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/43.0.2357.65 Mobile Safari/537.36'
+  name: 'Webview'
+  version: '43.0.2357.65'
```

Comment:
Is this the version of webview component, not Chrome?
The webview of iOS uses `Version/...`.
https://github.com/woothee/woothee/blob/master/testsets/smartphone_ios.yaml#L9